modules/mgmt-power-remediation-baremetal-about.adoc

```diff
@@ -105,6 +125,52 @@ spec:
 The `matchLabels` are examples only; you must map your machine groups based on your specific needs.
 ====
 
+.Sample `MachineHealthCheck` resource for bare metal, metal3-based remediation
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineHealthCheck
+metadata:
+  name: example <1>
```

Comment:
Seems to be missing a matching `<1>` callout description below?

Reply:
Thanks for your feedback @adellape . I have removed the <1>.